### DISCUSSION TITLE
Should I implement activitypub

### DISCUSSION THREAD

**Discussion by jovandeginste** [jovandeginste/workout-tracker#278] [Ideas]
This could enable sharing workouts and results, and pave the path to leader boards, without centralization.

**Comment by Monirzadeh** [comment]
it is intresting but share data should be super clear.
i think it is important that user can have completly privet  instants too.
or user can see his/her personal record in leader board of user that share publicly(he/she have option to not share data)

But i think important to have good test coverage first.

**Reply by jovandeginste** [reply]
I could not agree more: clear information about what is shared, and _not sharing_ by default.
